dev-python/pydata-sphinx-theme/pydata-sphinx-theme-0.3.1.ebuild

```diff
@@ -0,0 +1,22 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7,8} )
+
+DISTUTILS_USE_SETUPTOOLS=rdepend
+inherit distutils-r1
+
+DESCRIPTION="Bootstrap-based Sphinx theme from the PyData community"
+HOMEPAGE="https://github.com/pandas-dev/pydata-sphinx-theme"
+SRC_URI="https://github.com/pandas-dev/pydata-sphinx-theme/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD-with-disclosure"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="dev-python/sphinx[${PYTHON_USEDEP}]"
+
+# ModuleNotFoundError: No module named 'plotly' even if dev-python/plotly is installed
+#distutils_enable_sphinx docs dev-python/recommonmark dev-python/numpydoc dev-python/jupyter-sphinx dev-python/plotly
```

Comment:
I did not add the test because they introduce yet even more pytest-* packages. And this PR is already huge.

But, alright, I'll add them, I have the ebuilds lying around anyway.